### ISSUE TITLE
Support Library out of date

### ISSUE THREAD

**Issue by jkane001** [firebase/firebase-jobdispatcher-android#81]
Having a dependency on the support library means that my project now complains that there's a version inconsistency if I update to the latest version of the support library. So for the last couple weeks, I've had to leave an older version active in my code. Is the support library really necessary for this library (seems like it's mostly used for tests and annotations) - if it can be removed, that would be ideal.

Assuming that's not an option, may I request that it be updated to the latest versions as those are released?

Thanks!

**Comment by SUPERCILEX** [comment]
@jkane001 Few people seem to realize this, but gradle dependencies are like android styles, you can [override them](https://github.com/firebase/FirebaseUI-Android/issues/414#issuecomment-262283040).

TLDR; just include the latest version of `compile "com.android.support:support-annotations:$supportLibraryVersion"` and `compile "com.android.support:support-v4:${project.ext.supportLibraryVersion}"`.

**Comment by jkane001** [comment]
@SUPERCILEX true story, I did not know that; I was assuming I'd wind up with both versions fighting for supremacy. Thanks for the hint, I've updated my own dependencies, and all is well now.

**Comment by SUPERCILEX** [comment]
Awesome!
